docs/examples/llm/vllm.ipynb

```diff
@@ -0,0 +1,284 @@
+{
```

Comment:
<p>could you help add few more examples using different LLM models like one from <a href="https://docs.vllm.ai/en/latest/models/supported_models.html" target="_blank">https://docs.vllm.ai/en/latest/models/supported_models.html</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/run-llama/llama_index/pull/9257/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
Added examples for Mistral , Codellama and Orca models for completion .

Reply:
sweet thanks